src/Limenius/ReactRenderer/Twig/ReactRenderExtension.php

```diff
@@ -168,4 +177,20 @@ protected function shouldTrace($options)
     {
         return (isset($options['trace']) ? $options['trace'] : $this->trace);
     }
+
+    /**
+     * renderContext
+     *
+     * @return string a html with the context
+     */
+    protected function renderContext()
+    {
+        if ($this->needsRailsContext) {
+            $this->needsRailsContext = false;
```

Comment:
Nit: the other bool is called `needToSetContext`, maybe rename this one to `needToRenderContext`?